src/runtime/pbrt.mli

```diff
@@ -420,9 +437,13 @@ module Pp : sig
   val pp_int32 : formatter -> int32  -> unit
   (** [pp_unit fmt i] formats [i] value *)
 
+  val pp_unsigned_of_int32 : formatter -> [`unsigned of int32]  -> unit
```

Comment:
can you add a `@since 2.4` on both these functions please? This is going to warrant a release.